examples/openai-faiss-run.yaml

```
@@ -0,0 +1,83 @@
+# Example llama-stack configuration for OpenAI inference + FAISS (RAG)
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)